SourceryTests/Parsing/ComposerSpec.swift

```diff
@@ -396,6 +403,32 @@ class ParserComposerSpec: QuickSpec {
                     }
                 }
 
+                context("given generic inheritance") {
+                    it("resolves inherited type for subclasses") {
+                        let result = parse("""
+                            class Generic<T> {}
+                            class Specialized : Generic<Int> {}
+                        """)
+                        let generic = Class(name: "Generic", genericTypePlaceholders: [ GenericTypePlaceholder(placeholderName: TypeName("T")) ])
+                        let specialized = Class(name: "Specialized", inheritedTypes: ["Generic<Int>"])
```

Comment:
Can we also validate that `inherits` property is populated as expected and contains `Generic` type? Or this is not the case?

Reply:
Currently, this property is not populated. Reason is in `Composer.updateTypeRelationships(types:)` method `typesByName` contains "Generic" and "Specialized", however `type.inheritedTypes` contains "Generic<Int>", and therefore we cannot find "Generic", since we are searching for "Generic<Int>". 

I have a feeling it's a part of a bigger problem with generic names, but not sure how to resolve this one in scope of current pull request. Would love your advice on this.